services/form-definitions.ts

```diff
@@ -109,3 +111,62 @@ export async function createEmptyFormDefinition(
   `);
   return id;
 }
+
+export const findFormUsages = async (formId: string) => {
+  let results = [];
+  const queryString = `
+      PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+      PREFIX form: <http://lblod.data.gift/vocabularies/forms/>
+      PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+  
+      SELECT DISTINCT ?usage
+      WHERE {
+        ?form mu:uuid ${sparqlEscapeString(formId)}.
+        ?form form:targetType ?targetType .
+  
+        ?usage a ?targetType .
+      }
+    `;
+  try {
+    const queryResult = await query(queryString);
+    results = queryResult.results.bindings;
+  } catch (error) {
+    throw new HttpError(
```

Comment:
not sure if this actually adds much as we have the default error handler which does something similar with a general message (but this message is not usable in the app anyway so we can keep the general one i think?)